test/v1/models/request/sample/disclosures/AmendDisclosuresRequestBodySpec.scala

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.models.request.sample.disclosures
+
+import play.api.libs.json.{JsError, JsObject, Json}
+import support.UnitSpec
+import v1.models.request.disclosures.{AmendDisclosuresRequestBody, AmendTaxAvoidance}
+
+class AmendDisclosuresRequestBodySpec extends UnitSpec {
+
+  val model: AmendDisclosuresRequestBody = AmendDisclosuresRequestBody(Some(Seq(AmendTaxAvoidance("123","12-12"))))
+  val json = Json.parse(
+    """
+      |{
+      |   "taxAvoidance": [{"srn":"123","taxYear":"12-12"}]
```

Comment:
Can you bring tax year down and use a valid one for examples for those. Something like

  private val json = Json.parse(
    """
      |{
      |    "srn": "14211123",
      |    "taxYear": "2020-21"
      |}
    """.stripMargin
  )